SignalServiceKit/src/Messages/OWSMessageSender.m

```diff
@@ -501,6 +501,12 @@ - (void)sendMessageToService:(TSOutgoingMessage *)message
                 }];
             }
 
+            if (sendingRecipientIds.count < 1) {
+                // All recipients are already sent or can be skipped.
+                successHandler();
+                return;
```

Comment:
For example, if you create a group with no members, there's no need to send a message and this is not an error.

Reply:
Hmm... so what's the difference in behavior?

Originally:

- Create a group with no members
- Send
- Shows as failed

And now:

- Create a group with no members
- Send
- Shows as sent (but not delivered)

Is that right?

Reply:
It wasn't message sends to empty group that was failing, it was the "group update/creation" message itself that was failing.

Reply:
But it's worth noting that this will also affect a number of other edge cases.

* You're sending to a group with 1 member and you learn that that member has left the group between the time a message is enqueued and sent.
* You're sending to a group with 1 member and that member is blocked.

Reply:
I'll do my best to write a test plan that covers these and any other edge cases I can think of.

Reply:
I'm just trying to understand the new behavior, from the users perspective. 

What behavior does this change (in terms of what the user sees now?) And what did they see before this change?

Reply:
I guess you can cover that in the test plan - code changes LGTM

Reply:
Thanks for the clarifications!